fullstack/src/app/api/dust-data/route.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * @api {get} /api/dust-data Get dust data
+ * @description This file is the route for the dust-data API
+ */
+import db from "@/db";
+
+export const GET = async () => {
+    try {
+        const data = await db.any('SELECT * FROM dust');
```

Comment:
Are we sure this needs to be a select all?

Reply:
We will probably want a variety of selects possibly including a select all for raw data pages.
But we will likely also want to have selects where the time is in a specific timeframe instead off all data